config/routes.rb

```diff
@@ -9,6 +9,8 @@
     post "/unpublish-messages", to: "unpublish_messages#create"
 
     post "/notifications", to: "content_changes#create" # for backwards compaibility
+    post "/alert", to: "alert#create"
```

Comment:
```suggestion
    resources :alerts, only: %i[create]
```

I think this format of the route is better.

Reply:
Why do you think its better? I'm used to working on content-publisher where routes are defined as I've done above but am not too fussed either way

Reply:
Maybe it's more my personal opinion but I think it's neater, especially when a controller is a typical create, read, update, delete controller for a model.

I don't really mind in this case, but it does fit the pattern we've already got with `content_changes`, `spam_reports`, etc.

Reply:
Yeah true, but then further up there's the non-resource way of doing it. Maybe there just needs to be one pattern in the routes to settle it once and for all! & I've updated it to use the resource way. 😄